Adjust typography on Rpmlint and Build Result tabs

saraycp opened the pull request:
We changed the Rpmlint and Build Results partials for the package's build result box which appears in the request page as well.

We decided to change the font family of the Rpmlint result, that was in a `<pre>`tag, in order to reduce the number of different font families and increase readability.

Co-authored-by: Lukas Krause <lkrause@suse.de>

**Before:**
![Screenshot-2019-8-7 Request 8 (new)(5)](https://user-images.githubusercontent.com/2581944/62622537-e42fb500-b91e-11e9-8c89-97b08c6ab72c.png)
![Screenshot-2019-8-7 Request 8 (new)(4)](https://user-images.githubusercontent.com/2581944/62622546-e7c33c00-b91e-11e9-9612-b7f95380ad47.png)


**After:**

![Screenshot-2019-8-7 Request 8 (new)(2)](https://user-images.githubusercontent.com/2581944/62622466-b21e5300-b91e-11e9-81e3-22c623f611c4.png)
![Screenshot-2019-8-7 Request 8 (new)(3)](https://user-images.githubusercontent.com/2581944/62622475-b64a7080-b91e-11e9-809d-d7562f3a37ca.png)

DavidKang commented:
Just one thing, on the package view, the selectors of rpmlint are stacked
![Screenshot_2019-08-07 perl-Citrix](https://user-images.githubusercontent.com/1212806/62625856-5b1c7c00-b926-11e9-9965-139152d0ae9b.png)

krauselukas commented:
> Just one thing, on the package view, the selectors of rpmlint are stacked
> ![Screenshot_2019-08-07 perl-Citrix](https://user-images.githubusercontent.com/1212806/62625856-5b1c7c00-b926-11e9-9965-139152d0ae9b.png)

oh true, do you see an issue in that? or can we leave it as it is?

krauselukas commented:
> There is an issue in the build results on the package show:
> 
> ![Screenshot from 2019-08-07 15-08-26](https://user-images.githubusercontent.com/2650/62627899-915bfa80-b92a-11e9-99ac-d924993158f8.png)
> 
> Somehow, it happens only with this truck icon because it's a bit larger than the other icons like the cog and calendar.

@dmarcoux will have a look

krauselukas commented:
> There is an issue in the build results on the package show:
> 
> ![Screenshot from 2019-08-07 15-08-26](https://user-images.githubusercontent.com/2650/62627899-915bfa80-b92a-11e9-99ac-d924993158f8.png)
> 
> Somehow, it happens only with this truck icon because it's a bit larger than the other icons like the cog and calendar.

@dmarcoux we added some offset, this fixed the problem
![Screenshot-2019-8-7 ctris](https://user-images.githubusercontent.com/22001671/62632809-60cc8e80-b933-11e9-86c0-3eef2d2551b9.png)

dmarcoux commented:
It's much better now!

I realized that it's also a bit of an issue on mobile when the different sections/boxes are stacked:

![mobile-stacked-view](https://user-images.githubusercontent.com/1102934/62633360-4810a880-b934-11e9-89e8-07426c571c79.png)

To be honest, it was already an issue before as you can see in the screenshot. It's just a bit worse now.

krauselukas commented:
@dmarcoux we reduced the whitespace for xl screens, we got rid of the offsets and added a margin instead. We also adjusted the column length for certain screen sizes. 

There is still an issue in medium sized screens, which should be takled in another PR. The solution could be making it one column instead of two.

![Screenshot-2019-8-7 Request 4 (review)(3)](https://user-images.githubusercontent.com/22001671/62641255-86fa2a80-b943-11e9-8bf8-597438bfe5df.png)

vpereira commented on src/api/app/assets/stylesheets/webui2/build-results.scss:
```diff
@@ -1,6 +1,7 @@
-  font-size: .7rem;
+  font-size: .9em;
   max-height: 35rem;
+  font-family: $font-family-sans-serif;
```
in that case, because we want to simulate the output of a shell, IMO it would be ok to have a different font stack like:  

`font-family: Inconsolata, Consolas, monospace`

What do you think @hennevogel ? Do you want to have it look like a shell output?

vpereira replied:
however the original font-stack, `font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace; ` is as well good!

hennevogel replied:
Hm, I'm not sure that we want to simulate output of a shell. You are right, it's kind of a log. Maybe we need a component for things like this?

saraycp commented:
We moved back to monospace font family, but instead of having a tiny font size like the following

![tiny](https://user-images.githubusercontent.com/2581944/62705540-e280f280-b9ed-11e9-9c13-3ea8ef36001d.png)

we changed it to something bigger. We propose `0.95em` because with the combination `monospace -1em/16px` the text seems even bigger than the text in the description. Now we think it's more homogeneous.

![rmlint_monospace](https://user-images.githubusercontent.com/2581944/62705383-9635b280-b9ed-11e9-9883-2f77eec8d0b7.png)

What do you think @vpereira @hennevogel ?

**BTW, we'd suggest having `0.95em` also for the editor!! :nerd_face:**

saraycp commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Request: Review font classes / tags and line length](https://trello.com/c/gAEO50ru/1206-request-review-font-classes-tags-and-line-length)
